### PR TITLE
add ajax-edit,show new-stories

### PR DESCRIPTION
**Pull request by tunglam96hd**

### CODE REVIEW

**Comment by tunglam96hd** [comment]
https://edu-redmine.framgia.vn/issues/4990

**Comment by tunglam96hd** [comment]
ready

File: app/views/chapters/_chapter.html.erb [hunk 1]
```diff
@@ -3,12 +3,14 @@
...
       <%= link_to chapter.name, chapter %>
     </div>
-    <% if current_member.admin? %>
-      <div class="col-md-1 chapter">
-        <%= link_to chapter, method: :delete, data: {confirm: t(".you_sure")}, remote: true do %>
-          <span class="glyphicon glyphicon-trash"></span>
-        <% end %>
-      </div>
+    <% if logged_in? %>
+      <% if current_member.admin? %>
```
Comment (huuphuc93) [inline app/views/chapters/_chapter.html.erb]:
```suggestion
       <% if logged_in? && current_member.admin? %>
```

File: app/views/comments/update.js.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+$('#comment-<%= @comment.id %>').html('<%= j render 'comment', comment: @comment %>');
```
Comment (huuphuc93) [inline app/views/comments/update.js.erb]: render "comment" nhé

File: app/views/static_pages/_list_truyen.html.erb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <% collection.each do |story| %>
-  <div>
-    <%= link_to (image_tag story.image.url if story.image?), story %>
+  <div class="story">
+    <%= link_to (image_tag story.image.show.url if story.image?), story %>
```
Comment (huuphuc93) [inline app/views/static_pages/_list_truyen.html.erb]:
```suggestion
    <%= link_to story do %>
      <%= image_tag story.image.show.url if story.image? %>
    <% end %>
```

Comment (huuphuc93) [reply app/views/static_pages/_list_truyen.html.erb]: đoạn story.image.show.url dùng delegate đi em

**Comment by huuphuc93** [comment]
commented

**Comment by tunglam96hd** [comment]
ready

**Pull request merged**